### PR TITLE
Increase popup tutorial size

### PR DESCRIPTION
**Pull request by eatyourgreens**
- increase the height to `73vh`.
- increase the width to `400px`.
- remove the first heading to allow for more text on the first slide.
- change the slide image from `cover` to `contain`.
- includes #4639. Check the deployed branch with the DOM inspector. 
Page metadata in the DOM inspector, showing that the deployed ref is 'tutorial-sizing'.

The Daily Minor Planet poup tutorial with a wider (400px), taller (73vh) box, and the header image constrained to be square, with larger white margins to its left and right.

_Please request review from `@zooniverse/frontend` team or an individual member of that team._ 

## Package
lib-classifier

## Linked Issue and/or Talk Post
- towards #4558.

## How to Review
I think best to try out the popup and inline slide tutorial on a range of projects, to see how the size changes work with a variety of images and text.

Some random projects with a variety of tutorial content:
- Daily Minor Planet: https://local.zooniverse.org:3000/projects/fulsdavid/the-daily-minor-planet/classify/workflow/22354?env=production
- Beyond Borders: https://local.zooniverse.org:3000/projects/mainehistory/beyond-borders-transcribing-historic-maine-land-documents/classify/workflow/18383?env=production
- Planet Hunters Tess: https://local.zooniverse.org:3000/projects/nora-dot-eisner/planet-hunters-tess/classify/workflow/11235?env=production
- Deciphering Secrets: https://local.zooniverse.org:3000/projects/profdrrogerlouismartinez-davila/deciphering-secrets-unlocking-the-manuscripts-of-medieval-spain/classify/workflow/19276/subject-set/106993/subject/79761817?env=production
- SuperWASP: Black Hole Hunters: https://local.zooniverse.org:3000/projects/hughdickinson/superwasp-black-hole-hunters/classify/workflow/17386?env=production

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [x] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [x] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [x] Can submit a classification
- [x] Can sign-in and sign-out
- [ ] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Refactoring
- [x] The PR creator has described the reason for refactoring
- [x] The refactored component(s) continue to work as expected

### CODE REVIEW

**Comment by eatyourgreens** [comment]
Tests are broken because I introduced `useContext`, which isn't supported by Enzyme. Oh well, those tests need to be rewritten anyway.

**Comment by seanmiller26** [comment]
This change LGTM!

**Comment by lcjohnso** [comment]
I agree -- these changes look good.

As part of the review, can you highlight any significant differences in the display of tutorial images among the variety of current FEM projects linked at top of PR? Without a local or branch build, I would need some screenshots to assess the differences. Most likely the change is worth making, even if there are some effects to current projects.

**Comment by eatyourgreens** [comment]
>As part of the review, can you highlight any significant differences in the display of tutorial images among the variety of current FEM projects linked at top of PR? Without a local or branch build, I would need some screenshots to assess the differences. Most likely the change is worth making, even if there are some effects to current projects.

I've deployed commit 2ec3c0a so you can try it out here:
- Daily Minor Planet: https://fe-project-branch.preview.zooniverse.org/projects/fulsdavid/the-daily-minor-planet/classify/workflow/22354?env=production
- Beyond Borders: https://fe-project-branch.preview.zooniverse.org/projects/mainehistory/beyond-borders-transcribing-historic-maine-land-documents/classify/workflow/18383?env=production
- Planet Hunters Tess: https://fe-project-branch.preview.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/classify/workflow/11235?env=production
- Deciphering Secrets: https://fe-project-branch.preview.zooniverse.org/projects/profdrrogerlouismartinez-davila/deciphering-secrets-unlocking-the-manuscripts-of-medieval-spain/classify/workflow/19276/subject-set/106993/subject/79761817?env=production
- SuperWASP: Black Hole Hunters: https://fe-project-branch.preview.zooniverse.org/projects/hughdickinson/superwasp-black-hole-hunters/classify/workflow/17386?env=production

**Comment by eatyourgreens** [comment]
I've added the Github ref name to the page metadata, but it doesn't seem to be defined for branch builds. I'm probably missing a step in defining the build environment.
Screenshot of the page metadata, showing that zooniverse:deployed_commit is set but zooniverse:github_ref is empty.

**Comment by seanmiller26** [comment]
Is there a way to make the images edge-to-edge? As in - is there a certain aspect ratio the teams can use which will fill the whole area?

Will we be able to do anything for the teams who have formatted their images to fit the prior constraints? Sometimes, as a consequence of the change, the images are quite small. Could this be communication with the teams or another option?

Screen Shot 2023-05-16 at 9 31 35 AM

**Comment by eatyourgreens** [comment]
>Is there a way to make the images edge-to-edge? As in - is there a certain aspect ratio the teams can use which will fill the whole area?

The media box is 200px high, and now 400px wide, so 2:1, I think. 

Have we got any live project tutorials with video media? It’s worth testing those too. I imagine videos will simply expand to fill the available space.

**Comment by eatyourgreens** [comment]
Projects that might have video tutorials; Chimp & See and Manatee Chat.

**Comment by eatyourgreens** [comment]
I still can’t get through the Daily Minor Planet tutorial on my phone. This step has no navigation controls. 

A tutorial step on an iPhone XS, filling the full height of the phone screen, so that it’s impossible to move on to the next step.

File: .github/workflows/deploy_branch.yml [hunk 1]
```diff
@@ -58,7 +59,7 @@ jobs:
 
     - name: Modify & apply template
       run: |
-        sed 's/__IMAGE_TAG__/${{ github.sha }}/g' kubernetes/deployment-branch.tmpl | kubectl apply -f -
+        sed 's/__IMAGE_TAG__/${{ github.sha }}/g ; s/__REF_NAME__/${{ github.ref_name }}/g' kubernetes/deployment-branch.tmpl | kubectl apply -f -
```
Comment (eatyourgreens) [inline .github/workflows/deploy_branch.yml:62]: I’m not 100% sure I’ve got the `sed` syntax right here.

**Comment by lcjohnso** [comment]
@eatyourgreens  I confirmed with Sam that this PR is good to merge! Please have PR reviewed, approved, and merged at your discretion.

**Review by goplayoutside3 — APPROVED** [review]
Looks good!

**Pull request merged**